flutter_vlc_player/lib/src/vlc_player_controller.dart

```diff
@@ -120,11 +114,11 @@ class VlcPlayerController extends ValueNotifier<VlcPlayerValue> {
   /// This is a callback that will be executed once the platform view has been initialized.
   /// If you want the media to play as soon as the platform view has initialized, you could just call
   /// [VlcPlayerController.play] in this callback. (see the example)
-  final _onInit;
+  VoidCallback? onInit;
```

Comment:
can you make it more clear by providing a sample pseudo code?

Reply:
Here is an example of the `addOnInitListener` and `removeOnInitListener` methods: 
```dart
class VlcPlayerController {
  final List<VoidCallback> _onInitListeners = [];
  
  // ...

  void addOnInitListener(VoidCallback listener) {
    _onInitListeners.add(listener);
  }
  
  void removeOnInitListener(VoidCallback listener){
    _onInitListeners.remove(listener);
  }
  
   // ...
  
  _onInit(){
    _onInitListeners.forEach((listener) => listener());
  }
  
}
```

So once you call the `addOnInitListener`, it will be added to the internal `_onInitListeners` list and then called on vlc player initialized. To remove this listener, you should pass this listener to the `removeOnInitListener`, and then it will be deleted from the list of listeners and won't be called on player initialized. Please, review the [ChangeNotifier](https://api.flutter.dev/flutter/foundation/ChangeNotifier-class.html) class to get more info about this approach.